.github/workflows/ts_serialize_tests.yml

```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Deno lint
         run: deno lint
-      
```

Comment:
how are these in the project, am I missing some magic lint things?

Reply:
I'm not sure what you are asking here, what are you referring to?

Reply:
sorry my bad, some of the white space and quote changes

Reply:
I think my editor did that automatically, I can revert those changes if you want. I don't think we have a lint spec for `yml` files

Reply:
let's keep the changes, they're fine IMHO